examples/text/src/cross_text_selection/index.tsx

```diff
@@ -3,7 +3,7 @@
 // LICENSE file in the root directory of this source tree.
 
 import { root } from "@lynx-js/react";
-import { useEffect } from "react";
+import { useEffect } from "@lynx-js/react";
```

Comment:
Perhaps we could merge the two imports of `@lynx-js/react`.

```js
import { root, useEffect } from "@lynx-js/react";
```

Reply:
Done